cockatrice/src/carditem.cpp

```diff
@@ -16,6 +17,7 @@
 #include "settingscache.h"
 #include "tab_game.h"
 #include "pb/serverinfo_card.pb.h"
+#include "abstractcarditem.h"
```

Comment:
Abstractcarditem was needed to get the facedown status and QDebug was helping me to debug. No harm in having it in the file. Many files in the repo have it anyway.